.github/workflows/pr-staging-to-prod.yml

```diff
@@ -0,0 +1,61 @@
+name: 3 | Generate PR for releases/staging to releases/prod
+
+# This CI job is responsible for generating PRs that bring the HEAD of releases/staging into releases/prod.
+# These PRs are meant to be the only (standard) way that code is merged into the releases/prod branch.
+#
+# https://github.com/peter-evans/create-pull-request/blob/main/docs/examples.md#keep-a-branch-up-to-date-with-another
+
+on:
+  workflow_dispatch:
+
+jobs:
+  prod-gen-pr:
+    name: 'Generate PR for merging to releases/prod branch'
+    runs-on: ubuntu-latest
+    environment:
+      name: release
+    steps:
+      - uses: actions/checkout@7884fcad6b5d53d10323aee724dc68d8b9096a2e
```

Comment:
Why not use a newer version of checkout?

Reply:
Didn't you want to pin actions to specific commits before? However, in order to stay aligned with the rest, I would discuss whether pinning to `v3` or similar isn't more appropriate here

Reply:
All of the pinned versions are copied from the `mobile` repository. I could go either way on whether we want to pin to a newer commit hash. On one hand (1) newer versions are likely better but on the other hand (2) having more versions of the same action running in different UL repositories means there's more risk that we've pinned to a vulnerable version.

My $0.02 would be that it's fine to pin to a newer commit hash _but we don't want to just use a tag like `v3`_. Defer to your judgment here though.

Reply:
Makes sense, and overall I agree. I don't know whether we actively tack this right now, I remember there was a JIRA ticket, but one day I was working on pinning all the actions too. I think that's good move.

Reply:
All of the pinned versions are copied from the `mobile` repository. I could go either way on whether we want to pin to a newer commit hash. On one hand (1) newer versions are likely better but on the other hand (2) having more versions of the same action running in different UL repositories means there's more risk that we've pinned to a vulnerable version.

My $0.02 would be that it's fine to pin to a newer commit hash _but we don't want to just use a tag like `v3`_. Defer to your judgment here though.

Reply:
Sounds good - I'll pin to a newer version commit hash accordingly.